plugins/jenkins/test/models/samson/jenkins_test.rb

```diff
@@ -66,12 +90,19 @@ def stub_add_changeset
+  let(:jenkins_xml_configured) { "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<project>\n  <description>\n#### SAMSON DESCRIPTION STARTS ####\nFollowing text is generated by Samson. Please do not edit manually.\nThis job is triggered from following Samson projects and stages:\n* Project - Staging\nBuild Parameters starting with SAMSON_ are updated automatically by Samson. Please disable automatic updating of this jenkins job from the above mentioned samson projects before manually editing build parameters or description.\n#### SAMSON DESCRIPTION ENDS ####</description>\n  <keepDependencies>false</keepDependencies>\n  <properties>\n    <com.sonyericsson.rebuild.RebuildSettings plugin=\"rebuild@1.25\">\n      <autoRebuild>false</autoRebuild>\n      <rebuildDisabled>false</rebuildDisabled>\n    </com.sonyericsson.rebuild.RebuildSettings>\n  <hudson.model.ParametersDefinitionProperty><parameterDefinitions><hudson.model.StringParameterDefinition><name>SAMSON_buildStartedBy</name><description>Samson username of the person who started the deployment.</description><defaultValue/></hudson.model.StringParameterDefinition><hudson.model.StringParameterDefinition><name>SAMSON_originatedFrom</name><description>Samson project + stage + commit hash from github tag</description><defaultValue/></hudson.model.StringParameterDefinition><hudson.model.StringParameterDefinition><name>SAMSON_commit</name><description>Github commit hash of the change deployed.</description><defaultValue/></hudson.model.StringParameterDefinition><hudson.model.StringParameterDefinition><name>SAMSON_tag</name><description>Github tags of the commit being deployed.</description><defaultValue/></hudson.model.StringParameterDefinition><hudson.model.StringParameterDefinition><name>SAMSON_deployUrl</name><description>Samson url which triggered the current job.</description><defaultValue/></hudson.model.StringParameterDefinition><hudson.model.StringParameterDefinition><name>SAMSON_emails</name><description>Emails of the committers, buddy and user for current deployment. Please see samson to exclude the committers email.</description><defaultValue/></hudson.model.StringParameterDefinition></parameterDefinitions></hudson.model.ParametersDefinitionProperty></properties>\n  <scm class=\"hudson.scm.NullSCM\"/>\n  <canRoam>true</canRoam>\n  <disabled>false</disabled>\n  <blockBuildWhenDownstreamBuilding>false</blockBuildWhenDownstreamBuilding>\n  <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>\n  <triggers/>\n  <concurrentBuild>false</concurrentBuild>\n  <builders/>\n  <publishers/>\n  <buildWrappers/>\n</project>\n" }
```

Comment:
I am not stubbing using XML properly (like visual new lines) but xml is represented as string. Don't know if it worth the effort to convert like this? Your comments? Ruby client just returns a string.